.github/workflows/ci-valibot-validator.yml

```diff
@@ -19,6 +19,7 @@ jobs:
           node-version: 18.x
```

Comment:
How about updating it to `20.x`?

Reply:
Let's not change here in this PR for the reason of [the comment](https://github.com/honojs/middleware/pull/1067#pullrequestreview-2719920080).